From 651036c4c9f6e429306275103b6546a61a8362f7 Mon Sep 17 00:00:00 2001
From: Yang Yingliang <yangyingliang@huawei.com>
Date: Fri, 23 Sep 2022 22:18:03 +0800
Subject: [PATCH 105/109] Documentation: devres: add missing SPI helper

mainline inclusion
from mainline-v6.1-rc1
commit 49beeea7ebdb1d86ba8465e5021d72ad9e9474b8
category: bugfix
bugzilla: https://gitee.com/src-openeuler/kernel/issues/I8DDF5
Reference: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=49beeea7ebdb1d86ba8465e5021d72ad9e9474b8

----------------------------------------------------------------------

Add devm_spi_alloc_master() and devm_spi_alloc_slave() to devres.rst.
They are introduced by
commit 5e844cc37a5c ("spi: Introduce device-managed SPI controller allocation").

Signed-off-by: Yang Yingliang <yangyingliang@huawei.com>
Reviewed-by: Lukas Wunner <lukas@wunner.de>
Link: https://lore.kernel.org/r/20220923141803.75734-1-yangyingliang@huawei.com
Signed-off-by: Jonathan Corbet <corbet@lwn.net>

Conflicts:
	Documentation/driver-api/driver-model/devres.rst

Signed-off-by: Jiang Yi <jiangyi38@hisilicon.com>
---
 Documentation/driver-model/devres.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/driver-model/devres.txt b/Documentation/driver-model/devres.txt
index 43681ca0837f..c9b019c06f3b 100644
--- a/Documentation/driver-model/devres.txt
+++ b/Documentation/driver-model/devres.txt
@@ -392,6 +392,8 @@ SLAVE DMA ENGINE
   devm_acpi_dma_controller_register()
 
 SPI
+  devm_spi_alloc_master()
+  devm_spi_alloc_slave()
   devm_spi_register_master()
 
 WATCHDOG
-- 
2.23.0

